tools/sourcemap-tools/src/SourceProcessor.ts

```diff
@@ -155,7 +155,7 @@ export class SourceProcessor {
     public getSourceMapPathFromSource(source: string, sourcePath: string) {
         const match = source.match(/^\/\/# sourceMappingURL=(.+)$/m);
         if (!match || !match[1]) {
-            return Err('Could not find source map for source.');
+            return Err('could not find source map for source.');
```

Comment:
Shouldn't it be upper case?

Reply:
I'm trying to keep all errors lower case